src/Playwright/Core/Waiter.cs

```diff
@@ -73,15 +73,20 @@ public void Dispose()
                 dispose();
             }
 
+            var info = new Dictionary<string, object>
+            {
+                ["waitId"] = _waitId,
+                ["phase"] = "after",
+            };
+            if (!string.IsNullOrEmpty(_error))
+            {
+                info["error"] = _error;
```

Comment:
note: error was not allowed to have null, hence we need to change it. This was not caused by the roll afaik, just saw it in the protocol logs.